submissions/Sergey-Kali/index.html

```diff
@@ -0,0 +1,98 @@
+                            <p>Contacts</p>
+                        </a>
+                        <a href="#" class="popup-item"><img src="/images/tanslation.png" alt="Translate">
+                            <p>Translate</p>
+                        </a>
+                        <a href="#" class="popup-item"><img src="/images/videos.png" alt="Videos">
+                            <p>Videos</p>
+                        </a>
+                        <a href="#" class="popup-item"><img src="/images/search.png" alt="Search">
+                            <p>Search</p>
+                        </a>
+                        <a href="#" class="popup-item"><img src="/images/photos.png" alt="Photo">
+                            <p>Photo</p>
+                        </a>
+                        <a href="#" class="popup-item"><img src="/images/apps.png" alt="Apps">
+                            <p>Apps</p>
+                        </a>
+                        <a href="#" class="popup-item"><img src="/images/documents.png" alt="Document">
+                            <p>Document</p>
+                        </a>
+                        
+                    </div>
+                    <details>
+                        <summary class="more-button">More</summary>
+                        <div class="div-for-menu">
+                            <a href="#" class="popup-item"><img src="/images/chat.png" alt="Chat">
+                                <p>Chat</p>
+                            </a>
+                            <a href="#" class="popup-item"><img src="/images/classroom.png" alt="Classroom">
+                                <p>Classroom</p>
+                            </a>
+                            <a href="#" class="popup-item"><img src="/images/collection.png" alt="Collection">
+                                <p>Collection</p>
+                            </a>
+                            <a href="#" class="popup-item"><img src="/images/mars.png" alt="Mars">
+                                <p>Mars</p>
+                            </a>
+                            <a href="#" class="popup-item"><img src="/images/blogs.png" alt="Blogs">
+                                <p>Blogs</p>
+                            </a>
+                            <a href="#" class="popup-item"><img src="/images/promo.png" alt="Promos">
+                                <p>Promos</p>
+                            </a>
+
+                        </div>
+                        <a href="https://kottans.org" class="link">Even more from Kottans</a>
+                    </details>
+                </div>
+            </div>
+        </li>
+        <li class="list-item"><a href="#" class="bell"><img src="/images/bell.png" alt="Bell"></a></li>
+        <li class="list-item"><a href="#" class="user"><img src="/images/user.jpg" alt="Uset"></a></li>
+    </ul>
+   </nav>
+
+
+
+
+
+ 
+</body>
+</html>
```

Comment:
Add an empty line at the end of each file. [Reason](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline?noredirect=1&lq=1)